templates/aqua/0/rancher-compose.yml

```diff
@@ -0,0 +1,12 @@
+.catalog:
+  name: "aqua"
+  version: "2.6.3"
+  description: "aqua csp 2.6.3"
+  minimum_rancher_version: v1.1.0

```

Comment:
I don't know if this catalog entry can work on v1.1.0. But it is better to use the Rancher version which you test this entry with.

Reply:
Thanks for reminder, I corrected commit title now. Also we have already tested aqua on rancher 1.1.0